.buildkite/pipeline.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash -e
 
+echo "agents:
+    queue: m2-oss"
+echo "env:
+    JAVA_HOME: /opt/openjdk/openlogic-openjdk-17.0.9+9-mac-x64/jdk-17.0.9.jdk/Contents/Home"
+
```

Comment:
We can remove it completely. The 'agents' config are set in BK UI and affects all child steps. 
```suggestion
```